packages/api/src/router/user.ts

```diff
@@ -1,7 +1,17 @@
+import { TRPCError } from "@trpc/server";
 import { router, publicProcedure } from "../trpc";
 
 export const userRouter = router({
-  current: publicProcedure.query(({ ctx }) => {
-    return ctx.prisma.user.findUniqueOrThrow({ where: { id: ctx.user?.id } });
+  current: publicProcedure.query(async ({ ctx }) => {
```

Comment:
Is that a publicProcedure on purpose? 🤔

Reply:
I haven't put any real thought here, mostly just getting things working. It should probably be a private one though 👍